frontend/src/app/shared/components/dashboard/tweet-card/tweet-card.component.html

```diff
@@ -1 +1,29 @@
-<p>tweet-card works!</p>
+<div class="card d-flex rounded">
+  <div class="d-flex justify-content-between">
+    <div class="username m-3">
```

Comment:
.username class is not defined in any scss - thus does effect anything
remove it, to avoid confusion, keep your code clean

Some classes may be defined in external library (like Bootsrap), but not defined in your scss. Don't remove those.